lib/server.js

```diff
@@ -21,6 +23,33 @@ module.exports = async function({ plants, nurseries }) {
   }
   app.use(express.static(publicDir));
   app.use('/images', express.static(`${__dirname}/../images`));
+  app.use(express.json())
+
+  app.post('/get-vendors', async(req, res) =>{
+    const baseUrl = process.env.PAC_API_BASE_URL
+    const apiKey = process.env.PAC_API_KEY
+    const url = `${baseUrl}/Plant/FindVendorsForPlantName?plantName=${req.body.plantName}&zipCode=${req.body.zipCode}&radius=${req.body.radius}&limit=5`;
+    var response = await axios.get(url, {headers: {'Authorization': 'Bearer ' + apiKey}})
```

Comment:
Do not use `var`, it is deprecated, use `const` or `let` as appropriate to whether you will reassign.

Follow existing code style, objects should not be written all on one line.

Reply:
(I suggest adding eslint and a `.eslintconfig` file setting a standard for coding style to avoid a lot of extra work to format code and allow the use of `npx eslint . --fix` to clean up. `prettier` is also an option)